client/src/components/AdornedButton.js

```diff
@@ -5,15 +5,21 @@ import Button from "@material-ui/core/Button";
 import CircularProgress from "@material-ui/core/CircularProgress";
 import withStyles from "@material-ui/core/styles/withStyles";
 
-const styles = {
+const styles = theme => ({
     root: {
         marginLeft: 5
+    },
+    spinner: {
+        position: "relative",
+        marginLeft: "7px",
+        marginRight: "5px",
+        color: theme.palette.secondary.main
     }
-};
+});
 
-const SpinnerAdornment = withStyles(styles)(props => (
-    <CircularProgress className={props.classes.spinner} size={18} />
-));
+const SpinnerAdornment = withStyles(styles)(props => {
+    return <CircularProgress className={props.classes.spinner} size={18} />;
```

Comment:
The spinner class is now being applied, but root class not, could you please remove it completely then?